### ISSUE TITLE
Code review

### ISSUE THREAD

**Issue by kelomo2502** [teckim/portfolio#10]
- Code base looks good and very organised
- Naming of classes and sections looks very descriptive
- However, alt attribute for social icons are missing and could be added for better accessibility
